input/includes/GET_ServiceRequest.md

```diff
@@ -42,34 +42,6 @@ __Response__
         ]
       }
     },
-    {
```

Comment:
Do we know why this has been removed?

Reply:
Can be ignored, as I understand it the examples are based on tests and I guess it might not be the same test every time?

Reply:
From i understand the procedure, the examples are generated from tests as you say and whatever the script fetching the examples output we should use.